src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -390,19 +396,22 @@ public static void ValidateGlobalEndpointRouteConfig(RuntimeConfig runtimeConfig
         /// <param name="runtimeConfig"></param>
         public static void ValidateRestURI(RuntimeConfig runtimeConfig)
         {
-            // CosmosDB_NoSQL does not support rest. No need to do any validations.
-            if (runtimeConfig.DataSource.DatabaseType is DatabaseType.CosmosDB_NoSQL)
+            foreach (DataSource dataSource in runtimeConfig.DataSourceNameToDataSource.Values)
```

Comment:
This foreach loop doesn't make sense here. For example, if your datasources collection has 5 entries, you'd be validating the same `restPath` 5 times. It looks like the only valid check to include in the foreach would be to skip REST validation if any of the datasource DBtypes is Cosmos. Even then, we'd probably want a separate validation to ensure all database sources share the same dbtype, wouldn't we?